### PR TITLE
Updated farm-assets.rst

### PR DESCRIPTION
**Pull request by WW-TC**
Is there any modifications you would like me to do?

### CODE REVIEW

File: assets/item-asset/farm-asset.rst [hunk 1]
```diff
@@ -3,6 +3,20 @@
 Farm Assets
 ===========
 
-Temporarily noting this here, until farm assets are properly documented.
+**Grow** *ushort*: The item ID to spawn when harvested.
+
+**Grow_SpawnTable** *GUID*: GUID of a spawntable from which to spawn an item when harvested.
+
+**Growth** *uint*: How long before its fully grown. In seconds
```
Comment (MoltonMontro) [inline assets/item-asset/farm-asset.rst]: Move units (seconds) into sentence E.g., ``**Growth** *uint*: In seconds, how long before the crop is fully grown.``

File: assets/item-asset/farm-asset.rst [hunk 2]
```diff
@@ -3,6 +3,20 @@
 Farm Assets
 ===========
 
-Temporarily noting this here, until farm assets are properly documented.
```
Comment (MoltonMontro) [inline assets/item-asset/farm-asset.rst:6]: Maybe re-add this line to the top? You've documented all Farm asset properties, but these docs normally include a small lead/intro explaining the asset type, along with pertinent info from parent classes.

That's not a huge deal though -- it can just be added in a different PR. Just re-adding that line is fine, or a content block directive noting there's some info missing (for parent classes).

File: assets/item-asset/farm-asset.rst [hunk 3]
```diff
@@ -3,6 +3,20 @@
...
+
+**Growth** *uint*: How long before its fully grown. In seconds
 
 **Affected_By_Agriculture_Skill** *bool*: If true, the amount of crops acquired when harvesting the plant is affected by the Agriculture skill. Defaults to true.
+
+**Allow_Fertilizer** *bool*: If true, allows the player to use fertilizer to fully grow the plant. Defaults to true.
+
+**Ignore_Soil_Restrictions** *bool*: If false, only allow placement on Soil Materials. If true, allow placement anywhere. Default to false.
+
+**Harvest_Reward_Experience** *uint*: The amount of experience gained upon harvesting.
```
Comment (MoltonMontro) [inline assets/item-asset/farm-asset.rst]: Add ``Defaults to 1.`` to Harvest_Reward_Experience

File: assets/item-asset/farm-asset.rst [hunk 4]
```diff
@@ -3,6 +3,20 @@
...
+
+===========
+
+**NOTICE**: To have multi-harvest crops, use the following calcul "HP / 2 = Amount of possible harvests"
```
Comment (MoltonMontro) [inline assets/item-asset/farm-asset.rst]: When looking to add notes and similar comments, use a **content block directive** instead. These are preferable as they'll stand out more. Example below (note: I've removed the abbreviations as well):

```
.. note::
	
	To have crops that can be harvested multiple times, use the following calculation ``Health / 2 = Number of Harvests``.
```

**Comment by MoltonMontro** [comment]
Thank you for submitting a PR to help contribute! I've added some review comments. (A bit wordy, but they're minor.)

**Review by MoltonMontro — CHANGES_REQUESTED** [review]
Thank you for submitting a PR to help contribute! I've added some review comments. (A bit wordy, but they're minor.)

**Comment by WW-TC** [comment]
I've modified the file to add all the changes you requested. Is there anything else you need changed?

**Comment by MoltonMontro** [comment]
Looks good, thanks!

@SDGNelson, feel free to merge if it looks good to you too.

**Comment by SDGNelson** [comment]
Looks good, thanks for your contribution! This reminds me we should see about crediting contributors.
